package-res/resources/web/pentaho/util/logger.js

```diff
@@ -0,0 +1,112 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define(function () {
+
+  /**
+   * The `logger` namespace contains functions used to log messages in the console.
+   *
+   * @name logger
+   * @memberOf pentaho.util
+   * @namespace
+   * @amd pentaho/util/Logger
+   */
+  var logger =/** @lends pentaho.util.logger */{
+
+    /**
+     *  Property enumerating the various log levels.
+     *
+     *  @property logLevels
+     *  @type Array
+     */
+    logLevels: ["debug", "log", "info", "warn", "error", "exception"],
```

Comment:
You removed the function but forgot to remove the level. Remove the exception loglevel.